src/controllers/llmo/llmo-onboarding.js

```diff
@@ -538,6 +539,102 @@ export async function createOrFindOrganization(imsOrgId, context, say = () => {}
   return organization;
 }
 
+/**
+ * Toggles the www subdomain in a given URL.
+ * If the URL has www, it removes it. If it doesn't have www, it adds it.
+ * Only works for URLs without other subdomains.
+ * @param {string} url - The URL to toggle (e.g., "https://example.com" or "https://www.example.com")
+ * @returns {string} - The URL with www toggled
+ */
+function toggleWWW(url) {
+  try {
+    const urlObj = new URL(url);
+    const hostnameParts = urlObj.hostname.split('.');
+
+    if (urlObj.hostname.startsWith('www.')) {
+      urlObj.hostname = urlObj.hostname.replace('www.', '');
+    } else if (hostnameParts.length === 2) {
+      urlObj.hostname = `www.${urlObj.hostname}`;
+    }
+
+    return urlObj.toString();
+    /* c8 ignore next 3 */
+  } catch (error) {
+    return url;
+  }
+}
+
+/**
+ * Tests a URL against the Ahrefs top pages endpoint to see if it returns data.
+ * @param {string} url - The URL to test
+ * @param {object} ahrefsClient - The Ahrefs API client
+ * @param {object} log - Logger instance
+ * @returns {Promise<boolean>} - True if the URL returns top pages data, false otherwise
+ */
+async function testAhrefsTopPages(url, ahrefsClient, log) {
+  try {
+    const { result } = await ahrefsClient.getTopPages(url, 1);
```

Comment:
It's ok because we do it only once, but this [fetches more fields than we need](https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-ahrefs-client/src/index.js#L157-L159).

For anything that runs regularly, the [raw `sendRequest`](https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-ahrefs-client/src/index.js#L75) with hand-curated fields should be our go-to.